docs/sigridci-integration/github-actions.md

```diff
@@ -132,6 +132,12 @@ jobs:
           publishonly: true
         env:
           SIGRID_CI_TOKEN: "${{ secrets.SIGRID_CI_TOKEN }}"
+      - name: "Sigrid pull request feedback"
```

Comment:
I believe this shouldn't be added to this file but the one below this one for the pull request feedback 😉

Reply:
Whoops, you're completely right :)

Reply:
Looks all good now 😄